### DISCUSSION TITLE
How to change the mark from website format to yaml format

### DISCUSSION THREAD

**Discussion by josavion** [ultralytics/yolov5#13059] [Q&A]
My photo is website format there is about 9 thousand photo and I want to train the  Ai to learn so I just want to know it is necessary to change the mark to “yaml” format

**Comment by josavion** [comment]
This is my first time to use this website and i am a student from high school ,l start a project about  Campus bullying detection now I
Have some problem I help every one can help me thanks a lot.

**Comment by kronee0516** [comment]
you can take a look on this page first
https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/#4-train
btw you can find more information in the readme page in yolov5
